consistency/analysis/algorithm.ipynb

```diff
@@ -725,8 +725,8 @@
    "outputs": [],
    "source": [
     "import pandas as pd\n",
-    "\n",
-    "df = pd.read_csv (r'/home/gelaw/work-stuff/gocode/src/registry-experimental/consistency/rpc/google/cloud/apigeeregistry/v1/similarity/algorithms /vocab.csv')\n",
+    "path = \"path to vocab.csv\"\n",
```

Comment:
Is it possible to give a relative path here? 

If the directory structure is as follows:
```
-> consistency/analysis
       ->  algorithm.ipynb
       ->  vocab.csv
```

Then can we give a relative path like`pd.read_csv(r'vocab.csv')`?

Reply:
> > Then can we give a relative path like`pd.read_csv(r'vocab.csv')`?
> 
> Tried to do that but notebooks didn't seem to infer that on testing. Perhaps I will find some other workaround.

Okay, then I have aannother suggestion here, I see that you are expecting that user should set these paths correctly in the notebook, replace `path to vocab.csv` to the actual path.
 
Can you add a small section at the top of the notebook where users are supposed to set these paths in the beginning. and then run the notebook, that way they can set everything in the beginning and then playaround with the notebook without any issues.

Example, you can add a section like this at the top of the notebook:
```
text: 
Please set the absolute paths for the following files:
code:
# Absolute path for file "vocab.csv" 
vocab_csv_path = "PATH_TO_VOCAB_CSV"
...
```
And then use those variables in the rest of the notebook.